force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -322,17 +422,23 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer getSalary(String employeeName) {
         // Initialize a Map
-
+        Map<String, Integer> employeeSalary = new Map<String, integer>();
+            employeeSalary.put('John Doe', 50000);
+            employeeSalary.put('Jane Smith', 60000);
+            employeeSalary.put('Sam Brown', 55000);
+            employeeSalary.put('Alice Johnsones', 65000);    
+        
         // Add key-value pairs to the Map
         // add John Doe => 50000
         // add Jane Smith => 60000
         // add Sam Brown => 55000
         // add Alice Johnson => 65000
 
         // Get the salary of the employee
-
+        Integer salary = employeeSalary.get('Jane Smith');
+        System.debug('Salary of Employee: ' + salary);
         // Return the salary of the employee, or null if the employee is not found in the Map
-        return null; // Replace null with the variable you used to store the result
+        return employeeSalary; // Replace null with the variable you used to store the result
```

Comment:
You should be returning the value from the map that belongs to the key passed in as employeeName;